src/client/js/components/custom-select.js

```diff
@@ -8,6 +8,10 @@ const html = `
     color: var(--purple-70);
   }
 
+  :host([hidden]) {
```

Comment:
@mansaj inline style should only be used for emails, and we need a special CSP exception to allow it.  (Avoided for all other cases.)  

Here, hidden is an attribute, not really a style.  As such, it passes CSP.  The hidden attribute has great use cases including immediately indicating that the browser should not render the element – without having to wait for a stylesheet.  It sometimes has it's pitfalls, as it can be overwritten easily.  But used correctly it's great for accessibility, browser rendering, and element state.

In this _particular_ case, I've added this to the custom element in order to be sure it behaves as one would expect.  For example, another dev comes along and wants to add `<custom-select hidden></custom-select>`.